packages/chapters/src/templates/index-page.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { graphql } from 'gatsby'
+import { Hero, Layout, Content, ReadProgress } from 'common/components'
+import { Form } from '../components'
+
+export const IndexPageTemplate = ({ hero, modal }) => {
+  return (
+    <>
+      <Hero title={hero.title} />
+      <div dangerouslySetInnerHTML={{ __html: hero.subtitle }} />
```

Comment:
Remember the comment I mention to you in the other PR.
I think it would be more beneficial to use something like https://github.com/debtcollective/student-debt-campaign/blob/e9195e1414a611845e9fdd6605ebda6e630104ee/src/sections/Informative/index.js#L14 the advantages are just few but powerful:

- We create a practice for future developers to understand what is the suggested approach
- We support rich text from the CMS (allow to insert bold, italic, paragraphs and more from CMS)
- We don't use `dangerouslySetInnerHTML`

Reply:
Oh this looks good, but in this case, we need to write HTML with classes, is it possible to do it with markdown?